app/services/network_topology_service.rb

```diff
@@ -71,7 +71,7 @@ def build_entity_data(entity)
     data[:display_kind] = entity_display_type(entity)
 
     if entity.kind_of?(Host) || entity.kind_of?(Vm)
-      data[:provider] = entity.ext_management_system.name
+      data[:provider] = entity.try(:ext_management_system).try(:name)
```

Comment:
What about making this [consistent](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/services/infra_topology_service.rb#L47-L49) with the rest of the topology services?

```suggestion
      if entity.try(:ems_id)
        data[:provider] = entity.ext_management_system.name
      end
```